pymtl3/stdlib/stream/ifcs.py

```diff
@@ -24,11 +33,63 @@ def construct( s, Type ):
     s.val = InPort()
     s.rdy = OutPort()
 
+    s.MsgType   = Type
     s.trace_len = len(str(Type()))
 
+  def connect( s, other, parent ):
+
+    # SendCL (other) -> [ RecvCL -> SendRTL ] -> RecvRTL
+    if isinstance( other, CallerIfcCL ):
+      connect_valid = True
+      # Create an adapter and add to parents
+      m = RecvCL2SendRTL( s.MsgType )
+
+      if hasattr( parent, "RecvCL2SendRTL_count" ):
+        count = parent.RecvRTL2SendCL_count
```

Comment:
This branch is not covered by our regression tests. Did you mean `RecvCL2SendRTL_count` instead of `RecvRTL2SendCL_count`?